pkg/generators/models/testdata/cases/validation/expected/model_gender.go

```diff
@@ -0,0 +1,39 @@
+// This file is auto-generated, DO NOT EDIT.
+//
+// Source:
+//     Title: Test
+//     Version: 0.1.0
+package generatortest
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+)
+
+// Gender is an enum.
+type Gender string
```

Comment:
I guess we can actually simplify this by adding a `Validate()` method to the enum type right? The extra slices can still be useful though

Reply:
done